src/main/kotlin/com/github/thinkami/railroads/models/tasks/RoutesTask.kt

```diff
@@ -18,11 +19,14 @@ import com.intellij.openapi.wm.ToolWindowManager
 import org.jetbrains.plugins.ruby.gem.RubyGemExecutionContext
 import org.jetbrains.plugins.ruby.rails.model.RailsApp
 
-class RoutesTask(private val project: Project) : Task.Backgroundable(project, "task start...") {
+class RoutesTask(private val project: Project) : Task.Backgroundable(project, "Running rails routes") {
     private val module: Module = project.modules.first()
     private var output: ProcessOutput = ProcessOutput()
+    private var routes: List<BaseRoute> = listOf()
 
     override fun run(indicator: ProgressIndicator) {
+        indicator.isIndeterminate = true

```

Comment:
In large projects I would expect the bulk of the operations to be done in the `rails routes` portion, which means in practice the loading bar just hangs at 50% complete for a long time and appears like its "stuck"